testgrid/config/config.yaml

```diff
@@ -2676,78 +2591,45 @@ dashboards:
   - name: periodic-bazel-test-1.7
     test_group_name: periodic-kubernetes-bazel-test-1-7
 
+# These are the release *blocking* tests.  These provide a valid binary signal
+# to gate releases (alpha, beta, official).
+# This list is used by kubernetes/release/lib/releaselib.sh to provide go/nogo
+# signal for releases.
+# The first test/suite in the list is the primary suite for release purposes
 - name: release-1.7-blocking
```

Comment:
These looks like brand new variants, not just something to move into -blocking.  That seems to be outside the scope of this PR.  Post-this, anyone can add those new suites and put them into blocking.